storage/remote/queue_manager.go

```diff
@@ -249,7 +249,7 @@ func (t *QueueManager) Stop() {
 func (t *QueueManager) updateShardsLoop() {
 	defer t.wg.Done()
 
-	ticker := time.Tick(shardUpdateDuration)
+	ticker := time.NewTicker(shardUpdateDuration)
```

Comment:
Missing a defer ticker.Stop()

Reply:
Going to revert this one for now, as it breaks the code:

```
queue_manager.go:256:10: invalid operation: cannot receive from non-channel ticker (variable of type *time.Ticker)
```